packages/@webex/internal-plugin-feature/esbuild.config.js

```diff
@@ -0,0 +1,31 @@
+const esbuild = require('esbuild');
```

Comment:
why esbuild here ?

Reply:
Removed this, was a file i missed when duplicating the setup of `internal-plugin-flag` to test within another plugin.